components/audit_table.jsx

```diff
@@ -442,6 +442,8 @@ export function formatAuditInfo(audit, formatMessage) {
                 auditDesc = formatMessage(holders.attemptedLogin);
             } else if (userInfo[0] === 'success') {
                 auditDesc = formatMessage(holders.successfullLogin);
+            } else if (userInfo[0] === 'authenticated') {
+                // empty
```

Comment:
Nice find, @qichengzx! I'm wondering if we should add a new message, `holders.successfulAuthentication`:
```
    successfullAuthentication: {
        id: 'audit_table.successfullAuthentication',
        defaultMessage: 'Successfully authenticated,
    },
```
with the corresponding updates to https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json so that this message later gets translated. 

@jasonblais, perhaps you might have more context into the audit table requirements? With these changes, I think we end up with just a blank column which might be equally confusing. To be fair, it's not immediately obvious to me why authentication is audited separately from a successful login, but I'm guessing there's a way for it to fail after authentication that might be worth tracking.

Reply:
@lieut-data I think that makes sense to add. 

@qichengzx would you be interested helping with the suggested change?

Reply:
@jasonblais ok, I got it